integration-cli/docker_cli_help_test.go

```diff
@@ -15,7 +16,13 @@ func TestMainHelpWidth(t *testing.T) {
 
 	var home string
 	if runtime.GOOS != "windows" {
-		home = os.Getenv("HOME")
+		// Only do this on non-windows systems
+		home = homedir.Get()
+
+		// Don't check when HOME is /
+		if home == "/" {
+			home = ""
```

Comment:
Perhaps I misunderstood, but I thought you originally were saying that this test passed for you w/o this PR (on master), which didn't check for `/` so the if-statement should have generated a failure when $HOME is `/`.